FirebaseTestProject/Models/Post.swift

```diff
@@ -14,6 +14,7 @@ struct Post: FirebaseConvertable {
     let text: String
     let id: UUID
     let timestamp: Date
+    var isFavorite: Bool = false
```

Comment:
Not sure about adding this as a property to the `Post`. Perhaps we should consider going about this differently so that we don’t need a custom decoder—just an idea.

Reply:
I considered the same idea; I only came to this conclusion because it seems that a `Post` is or is not a favorite. It does add some overhead in the custom decoder.